gobblin-yarn/src/main/java/gobblin/yarn/GobblinApplicationMaster.java

```diff
@@ -147,18 +141,8 @@ public GobblinApplicationMaster(String applicationName, Config config) throws Ex
 
     this.serviceManager = new ServiceManager(services);
 
-    List<Tag<?>> tags = ImmutableList.<Tag<?>>builder()
```

Comment:
Actually we could not simply use `RootMetricContext`. First of all, there's no way to register non-context-aware metrics unless we have a `MetricRegistry` instance in `RootMetricContext` since `InnerMetricContext` does not allow registering standard metrics. But once we add a  `MetricRegistry` instance, then we have to worry about how to report them. Apparently the reporters in `RootMetricContext` only work with `ContextAwareMetric`s.